Moodis/Moodis/Ui/MenuForm.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using moodis;
+using Moodis.Network.Face;
+
+namespace Moodis.Ui
+{
+    public partial class MenuForm : Form
+    {
+        public static Boolean running = false;
```

Comment:
Is it necessary for this flag to be static, if we have an instance of MenuForm in Camera? Maybe it could be a simple public boolean value, since it won't lose its value until current instance of MenuForm is garbage collected